tests/integration/telemetry/stats/prometheus/stats.go

```diff
@@ -207,6 +211,45 @@ func TestStatsTCPFilter(t *testing.T, feature features.Feature) {
 		})
 }
 
+func TestStatsGatewayServerTCPFilter(t *testing.T, feature features.Feature) {
+	framework.NewTest(t).
+		Features(feature).
+		Run(func(t framework.TestContext) {
+			g, _ := errgroup.WithContext(context.Background())
+			for _, cltInstance := range client {
+				cltInstance := cltInstance
+				g.Go(func() error {
+					err := retry.UntilSuccess(func() error {
+						t.Logf("sending tcp traffic to gateway from sidecar")
+						requestURL := "curl --insecure -s -o /dev/null -w '%{http_code}' https://edition.cnn.com/politics"
```

Comment:
Once https://github.com/istio/istio/pull/39051 lands there is a much better way to call this, I will help set it up once that merges

Reply:
Figured would be easier to show vs tell: https://github.com/howardjohn/istio/tree/fetch-38495

Can you apply those changes/similar?

Reply:
really appreciate John!!!
updated.